lib/core.ps1

```diff
@@ -748,7 +756,7 @@ function env($name, $global, $val = '__get') {
 
     if ($val -eq '__get') {
         $RegistryValueOption = [Microsoft.Win32.RegistryValueOptions]::DoNotExpandEnvironmentNames
-        $EnvRegisterKey.GetValue($name, $null, $RegistryValueOption)
+        $EnvRegisterKey.GetValue($name, $null, $RegistryValueOption) + ""
```

Comment:
What's this `""` for?

Reply:
If the environment variable is empty we get a null exception later on. so we add the empty string to ensure it always returns a string even if the environment variable is empty.